tests/integration/components/task-card-test.js

```diff
@@ -4,22 +4,22 @@ import PageObject from 'ember-cli-page-object';
 import taskCardComponent from 'code-corps-ember/tests/pages/components/task-card';
 import moment from 'moment';
 import { Ability } from 'ember-can';
-import DS from 'ember-data';
 import Ember from 'ember';
+import DS from 'ember-data';
 import stubService from 'code-corps-ember/tests/helpers/stub-service';
 
 const { PromiseObject } = DS;
-const { RSVP, set, setProperties } = Ember;
+const { RSVP, setProperties } = Ember;
+const { set } = Ember;
 
 let page = PageObject.create(taskCardComponent);
 
 function renderPage() {
   page.render(hbs`
-    {{task-card
-      clickedTask=clickedTask
-      users=users
+    {{task-assignment
       task=task
       taskUser=taskUser
+      users=users
     }}`);
```

Comment:
This is the `task-card-test`, which means you're testing the `task-card` component, so the `renderPage` fuction should render that component.

The idea with integration tests is to ensure that the various bits of the application work as expected with each other. In `task-card-test`, we are testing how well the task card interacts with the outside and how well her internals work.

We want to write tests to ensure the internals that deal with task assignment work and the task card component interacts with the outside world correctly when those internals are used, so the old `renderPage()` was fine as it was.